ui/src/app/components/resource-actions-menu.tsx

```diff
@@ -16,12 +16,19 @@ export interface Action {
 export const ResourceActionsMenu = (props: {
   actions: Action[];
   disabled?: boolean;
-  menuButtonComponentOverride?: (props: { disabled: boolean }) => JSX.Element;
+  menuButtonComponentOverride?: (props: {
+    disabled: boolean;
+    title?: string;
+  }) => JSX.Element;
+  menuButtonTitle?: string;
 }) => {
-  const { actions, disabled, menuButtonComponentOverride } = props;
+  const { actions, disabled, menuButtonTitle, menuButtonComponentOverride } =
+    props;
 
-  const menuButtonComponent: (props: { disabled: boolean }) => JSX.Element =
-    menuButtonComponentOverride ?? SnowmanButton;
+  const menuButtonComponent: (props: {
+    disabled: boolean;
+    title?: string;
```

Comment:
This ensures that you can pass a title to an IconButton for accessibility and testability.

Reply:
Utilized in notebook-action-menu.